lib/functions/worker.sh

```diff
@@ -246,7 +249,7 @@ function LintCodebase() {
       elif [[ ${FILE_TYPE} == "CLOUDFORMATION" ]]; then
         LINT_CMD=$(
           cd "${WORKSPACE_PATH}" || exit
-          ${LINTER_COMMAND} "${FILE}" 2>&1
+          ${LINTER_COMMAND} "${FILE}" 2>&1 > >(tee "${LINT_OUTPUT_ONLY_FILE}")
```

Comment:
Apparently ordering matters (I still don't completely understand how the ordering works) --
```
> >(tee "${LINT_OUTPUT_ONLY_FILE}") 2>&1
```
will send both stderr and stdout to our temp file; whereas
```
2>&1 > >(tee "${LINT_OUTPUT_ONLY_FILE}")
```
does what we want